### DISCUSSION TITLE
How to show content in simple browser dynamically

### DISCUSSION THREAD

**Discussion by abhijeet-1110** [gitpod-io/openvscode-server#291] [Q&A]
I am running this project using this commands
tar -xzf openvscode-server-v1.63.0-linux-x64.tar.gz
cd openvscode-server-v1.63.0-linux-x64
./server.sh

In gitpod while the port is running it shows output in a simple browser. It is not working while i am running.

Can u help me need to change anything from my side.

**Comment by akosyakov** [comment]
Do you run on localhost or over https? Make sure to run in secure context, webviews are not working over http.

**Comment by abhijeet-1110** [comment]
@akosyakov over HTTPS. manually I am able to run, but I am not able to run dynamically.

My requirement is if I run nodejs project as 3000 it automatically launch simple browser like gitpod

**Comment by akosyakov** [comment]
Gitpod has custom VS Code extension which does it.
